### PR TITLE
Test case added for gcw-1851

### PR DESCRIPTION
**Pull request by abulsayyad123**
Hi,
This PR for test case of issue GCW-1851- https://jira.crossroads.org.hk/browse/GCW-1851.
Please review it.
Thanks

### CODE REVIEW

**Comment by namrataukirde** [comment]
Good work on test cases @abulsayyad123 👍

File: tests/acceptance/add-new-item-to-inventory-test.js [hunk 1]
```diff
@@ -19,16 +19,61 @@ module('Acceptance: Add item to inventory', {
     $.mockjax({url:"/api/v1/auth/current_user_profil*",
       responseText: data });
     location1 = FactoryGuy.make("location");
+    location2 = FactoryGuy.make("location");
     designation = FactoryGuy.make("designation", { state: "closed" });
     mockFindAll('designation').returns({json: {designations: [designation.toJSON({includeId: true})]}});
-    mockFindAll('location').returns({json: {locations: [location1.toJSON({includeId: true})]}});
+    mockFindAll('location').returns({json: {locations: [location2.toJSON({includeId: true})], meta: {search: location2.get('building').toString()}}});
```
Comment (Darshan-Chauhan) [inline tests/acceptance/add-new-item-to-inventory-test.js]: What's the use of passing **meta** parameter to mock request??

File: app/templates/items/new.hbs [hunk 2]
```diff
@@ -136,7 +136,7 @@
 
     <div class="{{if isMobileApp 'small-7 large-8' 'small-9'}} columns">
       {{#link-to "select_location"}}
-        <div class="{{if (is-or invalidLocation invalidScanResult) 'ember-view form__control form__control--error'}}">
+        <div class="{{if (is-or invalidLocation invalidScanResult) 'ember-view form__control form__control--error'}}" id="inventory_location">
```
Comment (Darshan-Chauhan) [inline app/templates/items/new.hbs]: Can we assign ID to input rather then to div?

Comment (abulsayyad123) [reply app/templates/items/new.hbs]: If in case value is not present in input we can fetch error value from child div.

**Pull request merged**